dask/dataframe/utils.py

```diff
@@ -160,3 +160,7 @@ def get_categories(df):
     if iscategorical(df.index.dtype):
         result['.index'] = df.index.categories
     return result
+
+
+def is_frame_or_series(x):
+    return isinstance(x, (pd.Series, pd.DataFrame))
```

Comment:
This is for preciseness and simplification. ``NDFrame`` matches higher dimensional data structures like ``Panel`` (3D).

```
issubclass(pd.Panel, pd.core.generic.NDFrame)
True
```